src/screens/Accounts.js

```diff
@@ -24,6 +24,7 @@ import CurrencyUnitValue from "../components/CurrencyUnitValue";
 import WhiteButton from "../components/WhiteButton";
 import CurrencyIcon from "../components/CurrencyIcon";
 import CounterValue from "../components/CounterValue";
+import PriceChange from "../components/PriceChange";
```

Comment:
My mistake here, but I think let's rename this to `DeltaChange`. because technically it's not about the price, component works with comparing anything.